python/lsst/ts/ess/sensors/socket_server.py

```diff
@@ -0,0 +1,173 @@
+# This file is part of ts_ess_sensors.
+#
+# Developed for the Vera C. Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+__all__ = ["SocketServer"]
+
+import asyncio
+import json
+import logging
+
+from .command_handler import CommandHandler
+
+
+class SocketServer:
```

Comment:
I suggest using ts_hexrotcomm's SingleSocketServer (assuming you only want to support one connection at a time, as most of our low-level controllers do0.

Reply:
Thanks for the suggestion. I will consider this in a future version.